plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/FieldEmbedding.kt

```diff
@@ -58,6 +59,7 @@ class UserFieldEmbedding(
 object ListSizeFieldEmbedding : FieldEmbedding {
     override val name = SpecialName("size")
     override val type = IntTypeEmbedding
+    override val viperType = Type.Ref
```

Comment:
This is a bit surprising; we don't want to use ints for the size here?

Reply:
I thought about it a bit, but the problem here is that this field is eventually going to be the real Kotlin field unlike the counter of function object calls or the result of `duplicable`.